test/BundleBulker.t.sol

```diff
@@ -152,6 +153,61 @@ contract BundleBulkerTest is Test {
             )
         );
     }
+
+    function test_DeflateInflator() public {
```

Comment:
recommend pulling this test out into `DeflateInflator.t.sol`

(i'll delete`DaimoTransferInflator` in a future PR, already superseded by `DaimoOpInflator`)